backend/connector_processor/exceptions.py

```diff
@@ -49,3 +51,10 @@ def __init__(self, core_err: ConnectorError) -> None:
         super().__init__(detail=core_err.message, core_err=core_err)
         self.default_detail = core_err.message
         self.status_code = 400
+
+
+class UnstractFSException(APIException):
+    default_detail = "Error testing connection. "
+
+    def __init__(self, detail: str = default_detail, status_code: Any = 500) -> None:
+        super().__init__(detail=detail, code=status_code)
```

Comment:
```suggestion
```
@kirtimanmishrazipstack this override isn't necessary. Please remove all similar overrides done for the APIException inheritance if we aren't doing anything different

Reply:
@chandrasekharan-zipstack if we don't override, then entire stack trace gets popped up in the UI.
I think this approch is better

Reply:
The entire stack trace to the user? Can you provide a screenshot?